test/unit/range/container/container_test.cpp

```diff
@@ -56,6 +58,20 @@ TYPED_TEST(container, construction)
     EXPECT_EQ(t3, t7);
 }
 
+TYPED_TEST(container, swap)
```

Comment:
Why move the test up? 😁

Reply:
because the constructors often already use swap :point_up: so it is better to test this before as it might resolve errors in the constructors on the fly